docs/guide/search/criteria_reference/createdat_criterion.md

```diff
@@ -0,0 +1,19 @@
+# CreatedAt Criterion
+
+The `CreatedAt` Search Criterion searches for products based on the date when they were created.
+
+## Arguments
+
+- `createdAt` - indicating the date that should be matched, provided as a `DateTimeInterface` object
```

Comment:
As a general rule I would avoid gerunds, so "indicating"->"indicates", "representing" -> "that represent", but these changes would have to be applied to all search criterion descriptions, therefore please treat this comment as a food for thought only.

Reply:
Yes, here it's for consistency, but on the next "big" change of all criteria pages we should rephrase this.